packages/apm/src/integrations/tracing.ts

```diff
@@ -283,6 +283,52 @@ export class Tracing implements Integration {
     });
   }
 
+  /**
+   * Returns a new Transaction either continued from sentry-trace meta or a new one
+   */
+  private static _getNewTransaction(hub: Hub, transactionContext: TransactionContext): Transaction {
+    let traceId;
+    let parentSpanId;
+    let sampled;
+
+    const header = Tracing._getMeta('sentry-trace');
```

Comment:
So I guess now if someone sets this in an SPA, all transactions would use this meta header right?

Reply:
Correct. For me this is opt-in behavior enough, this can't happen by accident so guarding this behind an option in the integration doesn't make a lot of sense to me.